com.unity.ml-agents/Runtime/Agent.deprecated.cs

```diff
@@ -1,63 +0,0 @@
-using System;
-using UnityEngine;
-using UnityEngine.Profiling;
-
-namespace Unity.MLAgents
-{
-    public partial class Agent
-    {
-        /// <summary>
-        /// Deprecated, use <see cref="WriteDiscreteActionMask"/> instead.
-        /// </summary>
-        /// <param name="actionMasker"></param>
-        [Obsolete("CollectDiscreteActionMasks has been deprecated, please use WriteDiscreteActionMask.")]
-        public virtual void CollectDiscreteActionMasks(DiscreteActionMasker actionMasker)
```

Comment:
Looks like there's still a reference to CollectDiscreteActionMasks
 in [`docs/Learning-Environment-Design-Agents.md`](https://github.com/Unity-Technologies/ml-agents/blob/main/docs/Learning-Environment-Design-Agents.md#iactuator-interface-and-actuatorcomponents)